projects/wave-core-new/src/lib/wave-core.module.ts

```diff
@@ -237,7 +239,7 @@ const WAVE_COMPONENTS = [
 ];
 
 @NgModule({
-    declarations: [...WAVE_PIPES, ...WAVE_COMPONENTS],
+    declarations: [...WAVE_PIPES, ...WAVE_COMPONENTS, LayerListMenuComponent],
```

Comment:
LayerListMenuComponent is already in WAVE_COMPONENTS

Reply:
b187362c5e0fdae195ec8e2a0550f3af3c36cc66